### PR TITLE
main: Auto-generate help for image types

### PR DESCRIPTION
**Pull request by cgwalters**
I forgot to add `vmdk` there.  This reduces the number of sources of truth for image types.

### CODE REVIEW

File: bib/cmd/bootc-image-builder/build_type.go [hunk 1]
```diff
@@ -39,3 +44,21 @@ func NewBuildType(imageTypes []string) (BuildType, error) {
...
+
+// allImageTypesString returns a comma-separated list of supported types
+func allImageTypesString() string {
+	keys := make([]string, 0, len(supportedImageTypes))
+	for k := range supportedImageTypes {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
+
+	r := ""
```
Comment (ondrejbudai) [inline bib/cmd/bootc-image-builder/build_type.go:56]: This loop can be just strings.Join(), see https://pkg.go.dev/strings#Join

**Review by ondrejbudai — CHANGES_REQUESTED** [review]
Great idea, one idea for simplification inline. :)

File: bib/cmd/bootc-image-builder/build_type.go (hunk 1 above)
Comment (cgwalters) [reply bib/cmd/bootc-image-builder/build_type.go:56]: Thanks, yep!  Originally I was looping over the map values, but then realized it'd be a good idea to sort.

Comment (cgwalters) [reply bib/cmd/bootc-image-builder/build_type.go:56]: This PR got rolled into https://github.com/osbuild/bootc-image-builder/pull/268

**Comment by cgwalters** [comment]
https://github.com/osbuild/bootc-image-builder/pull/268

**Pull request closed without merging**